sormas-e2e-tests/src/test/resources/features/sanity/web/Sample.feature

```diff
@@ -0,0 +1,22 @@
+@Sanity @Sample
+Feature: Create Sample
+
+  Scenario: Create a new Sample
+    Given I log in with the user
+    And I click on the Cases button from navbar
+    And I click on the NEW CASE button
+    When I create a new case with specific data
+    And I collect the case person UUID displayed on Edit case page
+    And I click on New Sample
+    When I collect the sample UUID displayed on create new sample page
+    And I create a new Sample with specific data
```

Comment:
I would split the tests related to create and edit.
there are two different functionality. we have to decide the approach here to have consistent test cases.

Reply:
What do you mean ? This is only one basic testcase of a sample creation, modification and checking the correct values on it. You would create a duplicate testcase just for sample creation just like this one but shorter ?

Reply:
exactly.

i would have a create scenario and an edit scenario.

at the moment we are using ui for creating stuff but in the future we will use an api function to create a sample. this can be used in the edit scenario by calling it via backend and then directly use the ui for editing.

Reply:
I think its not that long of a testcase and it also good to cover a basic sample creation with UI. 
API test cases hopefully will be used to create more complicated test cases like sormas2sormas, merge, importing, etc larger data that is cumbersome to do with UI.

Reply:
We will clarify this together

Reply:
The edit and create logic are separate and the steps can be reused however and wherever needed. If this needs clarification i agree with Tamas, we will take a decision together.